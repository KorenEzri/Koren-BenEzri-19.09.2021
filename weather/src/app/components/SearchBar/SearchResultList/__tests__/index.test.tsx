import * as React from 'react';
import { render } from '@testing-library/react';

import { SearchResultList } from '..';


describe('<SearchResultList  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<SearchResultList />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
