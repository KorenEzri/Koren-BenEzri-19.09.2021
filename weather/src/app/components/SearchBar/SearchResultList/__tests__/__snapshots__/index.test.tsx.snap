// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchResultList  /> should match snapshot 1`] = `
.c0 {
  list-style-type: none;
  position: absolute;
  width: 20%;
  margin-top: 45px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: -30px;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

@media (max-width:768px) {
  .c0 {
    margin-top: 55px;
  }
}

<ul
  class="c0"
/>
`;
